docusaurus/docs/Flutter/05-advanced/03-screen-sharing.mdx

```diff
@@ -85,19 +85,22 @@ After you create the extension, there should be a class called `SampleHandler`,
 
 ![Screen sharing dashboard](../assets/advanced_assets/broadcast-handler-implementation.png)
 
-:::tip
-If Xcode cannot find the `stream_video_screen_sharing` module add this code block to your app's Podfile file:
+To have access to our Handler implementation add `stream_video_screen_sharing` package to your app's `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  stream_video_screen_sharing: ^<latest_version>
+```
+
+Then for native code to see it, add it as a dependency manually for the extension target in the Podfile file:
 
 ```Podfile
-target 'ScreenSharing' do
+target 'YOUR_EXTENSION_NAME' do
   use_frameworks!
-  pod 'stream_video_flutter', :path => File.join('.symlinks', 'plugins', 'stream_video_flutter', 'ios')
+  pod 'stream_video_screen_sharing', :path => File.join('.symlinks', 'plugins', 'stream_video_screen_sharing', 'ios')
 end
 ```
 
```

Comment:
Maybe we can make this a bit clearer, what is the extension used for? What name do they put instead of the placeholder? etc